server/main.js

```diff
@@ -101,23 +103,24 @@ app.use(async (req, res, next) => {
     writeError(res, 400, `invalid method, only ${ALLOWED_METHODS} supported, you provided: ${req.body.method}`)
     return
   }
+  const msg = keccak256(req.body)
+  const address = verifyMessage(msg, auth)
```

Comment:
Message verification is a bit strange for ethers: https://docs.ethers.io/v5/api/utils/signing-key/#utils-verifyMessage

I need to test if this works as intended